cargo-shuttle/src/lib.rs

```diff
@@ -1701,8 +1702,19 @@ impl Shuttle {
             );
         }
 
+        // End this early for beta platform.
+        if client.beta {
```

Comment:
The flag is available in both places ;)
```suggestion
        if self.beta {
```

Reply:
Done.